src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -68,7 +68,7 @@ protected virtual string GetResourcePrefix(TypeInfo typeInfo)
                 throw new ArgumentNullException(nameof(typeInfo));
             }
 
-            return GetResourcePrefix(typeInfo, new AssemblyName(typeInfo.Assembly.FullName).Name, _resourcesRelativePath);
+            return GetResourcePrefix(typeInfo, GetRootNamespace(typeInfo.Assembly), GetResourcePath(typeInfo.Assembly));
```

Comment:
Why the change `GetResourcePath(typeInfo.Assembly)`? This is a change in behavior?

Reply:
Also, why bother changing the `baseNamespace` argument in a call when you're changing the called overload to ignore `baseNamespace`?

Reply:
I spoke with @dougbu in person and we agreed against his comments. @kichalla we changed to use GetResourcePath here because it's correct. Otherwise we would be ignoring the ResourceLocationAttribute in some cases.